pkg/cmd/bootstrap.go

```diff
@@ -123,9 +124,29 @@ func nonInteractiveMode(io *BootstrapParameters, client *utility.Client) error {
 	if err := checkMandatoryFlags(mandatoryFlags); err != nil {
 		return err
 	}
+	if io.GitHostAccessToken != "" {
+		err := setSecretIfNotSet(io.GitOpsRepoURL, io.GitHostAccessToken)
+		if err != nil {
+			return err
+		}
+		err = setSecretIfNotSet(io.ServiceRepoURL, io.GitHostAccessToken)
+		if err != nil {
+			return err
+		}
+	}
 	return nil
 }
 
+func setSecretIfNotSet(repoURL, accessToken string) error {
+	secret, _ := keyring.Get("kam", repoURL)
```

Comment:
This isn't handling the errors.

Also, it should probably be using the hostname from the URL, rather than the URL itself, as the token should be valid for all urls on the host.